src/components/adaptive-cards/Component/Component.jsx

```diff
@@ -118,6 +120,10 @@ export default function Component({data, className, style: styleProp}) {
         break;
       case acPropTypes.action:
       case acPropTypes.children:
+        childrenWithSpacers = value.flatMap((child) => [{type: 'Spacer', separator: child.separator, spacing: child.spacing || 'default'}, child]).slice(1);
```

Comment:
can't you define this variable here?
```js
const childrenWithSpacers = ...
```

Reply:
If I define it there, I get this eslint error:
"Unexpected lexical declaration in case block"